docs/source/command_line.rst

```diff
@@ -292,7 +292,7 @@ For more details, see :ref:`no_strict_optional`.
 
 ``--no-implicit-optional``
     This flag causes mypy to stop treating arguments with a ``None``
-    default value as having an implicit ``Optional[...]`` type.
```

Comment:
Is the referencing necessary here? It may be confusing when combined with the ellipsis in type brackets.

Reply:
I think the link is useful here. For someone who doesn't know `Optional` but is looking up this flag (maybe they inherited a script to run mypy), the link may be a lifeline. But let's remove the `[...]` part.